src/CMakeLists.txt

```diff
@@ -1,14 +1,19 @@
-# Version Updater library.
-add_subdirectory(version_updater)
+file(MAKE_DIRECTORY "${SM_SRC_DIR}/generated")
 
-# This part may move to a different file: it's the SM Data project.
+# Version Stub generation
 add_custom_command(
-  OUTPUT ${SM_SRC_DIR}/verstub.cpp
-  COMMAND VersionUpdater ${SM_SRC_DIR}/verstub.cpp
-  DEPENDS VersionUpdater
+  OUTPUT ${SM_SRC_DIR}/generated/verstub.cpp
+  COMMAND cmake
```

Comment:
Path issues here. I use the cmake gui program on Mac OS X, as specified by the docs.